server/routes/main.routes.js

```diff
@@ -4,6 +4,8 @@ const router = new Router();
 
 router.route('/songs').get(MainController.getSongs);
 
+router.route('/songs/:id').get(MainController.getSong);
```

Comment:
`getSong` returns an object containing all the metadata for a song. `getSongFromLibrary` returns a stream of a songs music.